crates/api_common/src/site.rs

```diff
@@ -124,18 +124,25 @@ pub struct ResolveObjectResponse {
 #[cfg_attr(feature = "full", ts(export))]
 /// Fetches the modlog.
 pub struct GetModlog {
+  /// Filter by the moderator.
   #[cfg_attr(feature = "full", ts(optional))]
   pub mod_person_id: Option<PersonId>,
+  /// Filter by the community.
   #[cfg_attr(feature = "full", ts(optional))]
   pub community_id: Option<CommunityId>,
+  /// Filter by the modlog action type.
   #[cfg_attr(feature = "full", ts(optional))]
   pub type_: Option<ModlogActionType>,
+  /// Filter by listing type. When not using All, it will remove the non-community modlog entries.
```

Comment:
This is a bit vague. I think you mean admin actions (eg site ban or purge).

Reply:
K, updated with a few examples.